notes-webapp/src/main/webapp/vue-app/notes-editor/components/NotesEditorDashboard.vue

```diff
@@ -461,7 +461,7 @@ export default {
         // if draft page not created persist it only the first time else update it in browser's localStorage
         if (this.note.draftPage && this.note.id) {
           this.note.parentPageId = this.parentPageId;
-          localStorage.setItem(`draftNoteId-${this.note.id}`, JSON.stringify(this.note));
+          localStorage.setItem(`draftNoteId-${this.note.id}`, JSON.stringify(draftNote));
```

Comment:
can you explain here why you change this.note to draftNote ?

Reply:
> can you explain here why you change this.note to draftNote ?

this.note does not have  the last up-to-date content of ckeditor.